scripts/docker/spark-cluster-managers/spark_yarn_cluster/Dockerfile

```diff
@@ -52,7 +52,7 @@ ENV HADOOP_MAPRED_HOME /usr/local/hadoop
 ENV HADOOP_YARN_HOME /usr/local/hadoop
 ENV HADOOP_CONF_DIR /usr/local/hadoop/etc/hadoop
 
-RUN sed -i '/^export JAVA_HOME/ s:.*:export JAVA_HOME=/usr/lib/jvm/jre-1.7.0-openjdk.x86_64\nexport HADOOP_PREFIX=/usr/local/hadoop\nexport HADOOP_HOME=/usr/local/hadoop\n:' $HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+RUN sed -i '/^export JAVA_HOME/ s:.*:export JAVA_HOME=/usr/lib/jvm/jre-1.7.0-openjdk-1.7.0.201-2.6.16.1.el7_6.x86_64\nexport HADOOP_PREFIX=/usr/local/hadoop\nexport HADOOP_HOME=/usr/local/hadoop\n:' $HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
```

Comment:
`1.7.0-openjdk-1.7.0.201-2.6.16.1.el7_6.x86_64\` might not work - this changes when `yum install -y java-1.7.0-openjdk-devel` is picking up a newer version

Reply:
In my `/usr/lib/jvm`, there are `jre`, `jre-1.7.0`, `jre-1.7.0-openjdk`, `jre-1.7.0-openjdk-1.7.0.201-2.6.16.1.el7_6.x86_64`, `jre-openjdk`(but no `jre-1.7.0-openjdk.x86_64`), and it seems the contents of them are all the same.
So should I change it to simply `jre`, or anything else?

Reply:
what I mean is that part `.201-2.6.16.1.el7_6.` changes for each release

Reply:
I use the path `/usr/lib/jvm/jre-1.7.0-openjdk` instead, is that ok?

Reply:
sounds better if it works